dashboard/OWNERS

```diff
@@ -0,0 +1,14 @@
+approvers:
+  - jlewi
+  - wbuchwalter
+  - Jimexist
+reviewers:
```

Comment:
Remove all of these reviewers.
Approvers shouldn't also be listed as reviewers.

None of the non-approvers listed here have special familiarity with react/javascript. 
I sent a request yesterday for more people with javascript/react experience to volunteer but we'll see.

You could add @swiftdiaries as a reviewer. He's working on the central UI.